lib/parsers/asm-parser-dotnet.ts

```diff
@@ -38,13 +38,15 @@ export class DotNetAsmParser implements IAsmParser {
         const allAvailable: string[] = [];
         const usedLabels: string[] = [];
 
-        const methodRefRe = /^\w+\s+\[(.*)]/g;
-        const tailCallRe = /^tail\.jmp\s+\[.*?](.*)/g;
+        const methodRefRe = /^(call|jmp|tail\.jmp)\s+(.*)/g;
         const labelRefRe = /^\w+\s+.*?(G_M\w+)/g;
 
         for (const line in asmLines) {
-            const trimmedLine = asmLines[line].trim();
+            let trimmedLine = asmLines[line].trim();
```

Comment:
It might be better to replace this with a regex now. Something like:
```ts
const removeCommentAndWsRe = /^\s*(?<line>.*?)(\s*;.*)?\s*$/; // outside the loop
//...
const trimmedLine = removeCommentAndWs.exec(asmLines[line])?.groups?.line;
```

we then drop lines 47-49


(though regex-fu isn't well represented in this file, admittedly)

Reply:
Done.